core-bundle/assets/controllers/image-size-controller.js

```diff
@@ -27,10 +33,10 @@ export default class extends Controller {
     }
 
     disconnect () {
-        this.element.parentNode.classList.remove('wizard');
-        this.select.removeEventListener('change', this.updateWizard);
-        this.buttonImage.remove();
-        this.button.remove();
+        this.element.parentNode?.classList.remove('wizard');
+        this.select?.removeEventListener('change', this.updateWizard);
```

Comment:
Could we may be move the event listener also to the HTML by adjusting the `ImageSize` widget?

Reply:
When we would implement it from scratch, we not use dynamically injected elements, I think. But right now I would leave it as it is.

Reply:
We don't really need to inject the button via JS. It's now always added anyway - so it might as well be output by the Widget directly.

Reply:
I agree. Do you want to make the according changes? :relaxed: You can push directly on this branch.

Reply:
See e1003d8dcc2e88eeaf84976c0d670026e79fb831

Feel free to revert if too intrusive of a change.

Reply:
Not at all! I like it :heart: